README.md

```diff
@@ -48,62 +52,48 @@ This is what Timber's `.twig` files look like (from this [Hello World example](h
 
 Once Timber is installed, it gives any WordPress theme the ability to take advantage of the power of Twig and other Timber features.
 
-### Looking for docs?
+## Documentation
+
 * [Timber Documentation](https://timber.github.io/docs/)
+* [Timber 2.0 Upgrade Guide](https://timber.github.io/docs/v2)
```

Comment:
```suggestion
* [Timber 2.0 Upgrade Guide](https://timber.github.io/docs/v2/upgrade-guides/2.0/)
```

Reply:
Better to link to the actual guide.

Once v2 will go live, will the docs still live under the /v2/ slug? And do we need to prepare for that?